cliff_detector_node:
  ros__parameters:
    point_cloud_topic: "/zed/zed_node/point_cloud/cloud_registered"
    camera_z_offset: 1.3 #1.3
    transform_tolerance: 0.01
    ground_level: -0.4  #w.r.t. base_footprint
    angle_min: -3.1415927410125732 #-1.5708,  # -M_PI/2
    angle_max: 3.1415927410125732 #1.5708,  # M_PI/2
    angle_increment: 0.017501909285783768  #0.0087,  #S M_PI/360.0
    scan_time: 0.3333
    range_min: 0.4
    range_max: 4.0
    use_inf: True
    inf_epsilon: 1.0


